desktop/extensions-sdk/dev/test-debug.md

```diff
@@ -2,16 +2,18 @@
 title: "Step three: Test and debug"
 description: Step four in the extension creation process
 keywords: Docker, Extensions, sdk, preview, update, Chrome DevTools
+redirect_from:
+- /desktop/extensions-sdk/build/test-debug/
 ---
 
-Once your extension has been [built and installed](build-install.md), you can preview your extension in Docker Desktop. 
+Once your extension has been [built and installed](../build/frontend-extension-tutorial.md), you can preview your extension in Docker Desktop. 
 
 ## Preview the extension
 
 To preview the extension in Docker Desktop, close and open Docker Dashboard once the installation is complete.
 
 The left-hand menu displays a new tab with the name of your extension. 
-The example below shows the [`Min FrontEnd Extension`](set-up/minimal-frontend-extension.md). 
+The example below shows the [`Min FrontEnd Extension`](../build/minimal-frontend-extension.md). 
 
 ![minimal-frontend-extension](images/ui-minimal-extension.png)
```

Comment:
This section could be either moved to minimal-frontend-extension.md, or removed if it's a duplicate, but it's not relevant in this page

Reply:
The intro in this page needs to be updated to make sense as a standalone section under SDK tools

Reply:
Should be better in [ae6e58b](https://github.com/docker/docs/pull/16290/commits/ae6e58bc200fdb9fd9bfa38be30d43b0917fc0ec)